ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -95,6 +98,9 @@ public String getVersion() {
 
             applyRAs(crac, network, newRemedialActionsResultList);
             tempSensitivityAnalysisResult = SystematicSensitivityAnalysisService.runAnalysis(network, crac, computationManager);
+            if (tempSensitivityAnalysisResult.getStateSensiMap().containsValue(null)) {
+                return CompletableFuture.completedFuture(new RaoComputationResult(RaoComputationResult.Status.FAILURE));
```

Comment:
you could put a break instead of returning a failure, so we return the best result we found so far